src/test/kotlin/uk/gov/dluhc/printapi/service/PrintServiceTest.kt

```diff
@@ -47,4 +56,23 @@ class PrintServiceTest {
         verify(printDetailsMapper).toPrintDetails(message, ero, localAuthority.name)
         verify(printDetailsRepository).save(printDetails)
     }
+
+    @Test
```

Comment:
Separate tests - I like it. Will make it easier to pull out the Dynamo stuff shortly 👍

Reply:
Yeah, where possible done this - but in many cases the existing tests failed with additional code so those tests were updated and have a mix of MySQL and DynamoDB mocks and verifies.